src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const populationList = [...document.querySelectorAll('.population')];
+const totalSpan = document.querySelector('.total-population');
+const averageSpan = document.querySelector('.average-population');
+
+function getNumberFromString(str) {
+  return +str.split(',').join('');
+}
+
+function thousandsSeparator(num) {
+  const source = '' + num;
+  const result = [];
+
+  for (let i = source.length; i > 0; i -= 3) {
+    result.unshift(source.substring(i - 3, i));
+  }
+
+  return result.join(',');
+}
```

Comment:
You can simple use `num.toLocaleString('en')` instead your method:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString